NorthstarDLL/squirrel/squirrel.cpp

```diff
@@ -255,6 +255,14 @@ template <ScriptContext context> void SquirrelManager<context>::VMCreated(CSquir
 
 		defconst(m_pSQVM, pair.first.c_str(), bWasFound);
 	}
+
+	auto loadedPlugins = &g_pPluginManager->m_vLoadedPlugins;
+	for (const auto& pluginName : g_pModManager->m_PluginDependencyConstants)
+	{
+		auto f = [&](Plugin plugin) -> bool { return plugin.dependencyName == pluginName; };
+		defconst(m_pSQVM, pluginName.c_str(), std::find_if(loadedPlugins->begin(), loadedPlugins->end(), f) != loadedPlugins->end());
```

Comment:
Can we scope this somehow, or maybe even define a function to check if a plugin is defined?

Reply:
so you want this to be moved into it's own function?

Reply:
I'm wondering if there's a better way to do this without polluting the global namespace with arbitrarily-named constants.

Reply:
I don't understand. there is no global variable?

Reply:
Oh, nevermind, I misinterpreted the code and didn't look at the sample mod as I was on my phone.